nisystemlink/clients/file/models/_file_metadata.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+from typing import Dict, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+from pydantic import Field
+
+from ._link import Link
+
+
+class FileMetadata(JsonModel):
+    field_links: Optional[Dict[str, Link]] = Field(None, alias="_links")
+    """
+    The links to access and manipulate the file:
+    - data: Link to download the file using a GET request
+    - delete: Link to delete the file using a DELETE request
+    - self: Link to the file's metadata
+    - updateMetadata: Link to update the file's metadata using a POST request
+    """
+    created: Optional[datetime] = Field(None, example="2018-05-15T18:54:27.519Z")
```

Comment:
I think our style guide recommends having spaces between fields. I would expect to have a blank line before each here.